card/mod/core/set/all/actify.rb

```diff
@@ -128,3 +128,11 @@ def with_transaction_returning_status
 def success
   Env.success(cardname)
 end
+
+def name_before_act
+  name_before_last_save || name_was
+end
```

Comment:
perhaps a comment here?  guessing behavior changed to something a bit more surprising?